dev/bench/gitlab-bench.yml

```diff
@@ -1,6 +1,7 @@
 
 bench:
-  stage: stage-1
+  stage: build
+  needs: []
```

Comment:
Adding the `needs: []` line means that this job will not have to wait for the Docker image build to run before being started (cf. behavior of `pkg:nix` jobs at https://gitlab.com/Zimmi48/coq/-/pipelines/345616842). This is mostly irrelevant since this initial stage is usually skipped in the main repo and this is a manual job, but what this change intended?

Reply:
Yes, the bench doesn't need docker.